endpoints/auth.go

```diff
@@ -562,7 +563,8 @@ func CurrentUser(c Context, scopes []string, audiences []string, clientIDs []str
 }
 
 func init() {
-	if appengine.IsDevAppServer() {
+	// TODO(jbd): Replace it with appengine.IsDevAppServer when it's implemented.
+	if os.Getenv("RUN_WITH_DEVAPPSERVER") != "" {
```

Comment:
Even if this is not really changing anything I'd merge this if the new code works for both worlds.